gensrc/thrift/FrontendService.thrift

```diff
@@ -689,4 +708,6 @@ service FrontendService {
     TStreamLoadPutResult streamLoadPut(1: TStreamLoadPutRequest request)
 
     Status.TStatus snapshotLoaderReport(1: TSnapshotLoaderReportRequest request)
-}
+
+    TBootstrapResult ping(1: TFrontendPingFrontendRequest request)
```

Comment:
The request and response names are unmatched with the service name.

Reply:
thank you for you advice, I will change it.

Reply:
If service is named as `ping`, why not just name the request/response as `TPingRequest`/`TPingResponse`.

Reply:
Sorry to intervene, It's just my personal options, you can just ignore them all ^_^

Reply:
In my option, by TFrontendPingFrontendRequest, we can know the request is sent by fe to fe, by TPingRequest is less information?

Reply:
I get what you mean, actually,   TFrontendPingFrontendRequest is like the name in   
```
 TBrokerOperationStatus ping(1: TBrokerPingBrokerRequest request);
```